### DISCUSSION TITLE
[OOMKilled] job failed because of Insufficient memory

### DISCUSSION THREAD

**Discussion by v-ccross** [PaddlePaddle/PaddleSpatial#195] [BaiduKDDCup2022]
Hi, running into an issue with job failing due to insufficient memory. Could I see more details as to where and how this issue occurs? Team name is v-ccrossTeam. Submission time was 2022-06-19 19:35.

**Comment by Chaodong** [comment]
same problem

**Comment by makwingchi** [comment]
+1
